docs_user/assemblies/assembly_storage-requirements.adoc

```diff
@@ -10,7 +10,7 @@ When looking into the storage in an {rhos_prev_long} ({OpenStackShort}) deployme
 two kinds, the storage requirements of the services themselves and the
 storage used for the {OpenStackShort} users that the services will manage.
```

Comment:
Does "storage" refer to a specific type of storage, or is this an all encompassing term for all the types of storage?

Reply:
The storage used by users can be any type of storage: Block, Shares, Objects.
And within each type there can be different transport protocols and technologies.
What this paragraph is trying to point out is that talking about "storage" means 2 things: the storage needed for the service to run and the storage the service manages.